src/lib/helpers.ts

```diff
@@ -304,3 +304,18 @@ export async function generatePKCEChallenge(verifier: string) {
   const hashed = await sha256(verifier)
   return base64urlencode(hashed)
 }
+
```

Comment:
It's highly unlikely that someone is using the method but leaving `generatePKCEChallenge` and `generatePKCEVerifier` as exported as I'm not sure